go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v1.0.0
+	github.com/zalando/go-keyring v0.1.0
```

Comment:
I can see the new none commit changes in go.mod file when I execute ```go mod tidy``` on your changes.

<img width="557" alt="Screen Shot 2020-11-11 at 12 24 04 PM" src="https://user-images.githubusercontent.com/19851115/98778789-e08bb780-2418-11eb-8aef-aa6fbfcd3ae8.png">

Due to this go.mod is captured in another validation check statement - https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-developer_kam/80/pull-ci-redhat-developer-kam-master-unit/1326179715039367168#1:build-log.txt%3A251

Can you please run ```make gomod_tidy``` and commit the changes. 

This will fix the openshift CI test failure for doc validation step.

Reply:
I have created a pr https://github.com/redhat-developer/kam/pull/81 for validation sync